src/vibrationRe.rei

```diff
@@ -0,0 +1,3 @@
+let vibrate: pattern::array int => repeat::bool => unit;
+
+external cancel : unit => unit = "" [@@bs.scope "Vibrrate"] [@@bs.module "react-native"];
```

Comment:
In other interface files the convention is that we hide externals with `let`s. Do you like the convention?